source/02_algo/imitation_learning_en.rst

```diff
@@ -0,0 +1,81 @@
+
+The current research hotspots of BC algorithms mainly focus on two aspects: meta-learning and behavior cloning using VR devices.
+
+
+IRL
+~~~~~~~~
+
+The main goal of IRL is to solve the problem of finding enough high-quality data during data collection. Specifically, IRL first learns a reward function from expert data, and then uses this reward function for subsequent RL training. With such an approach, IRL can theoretically outperform expert data.
+
+From the specific work above, Ziebart et al. [4] first proposed the maximum entropy IRL, which utilizes the maximum entropy distribution to obtain good prospects and effective optimization. Later in 2016, Finn et al. [5] proposed a model-based approach to IRL called Guided Cost Learning (guided cost learning), this method uses a neural network to represent the cost to improve the expressive ability. Subsequently, Hester et al. proposed DQfD [6], which requires only a small amount of expert data, and significantly speeds up training through the pre-training startup process and subsequent learning process. Later methods such as T-REX [7] proposed a structure based on ranking expert data, which indirectly learned the reward function by comparing which expert data performed better.
+
+
+Adversarial Structured IL
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The main goal of the Adversarial Structured IL approach is to address the efficiency of IRL. It can be seen from the algorithm of IRL that even if it learns a very good reward function, since the final strategy still needs to perform reinforcement learning steps, if the strategy can be learned directly from expert data, the efficiency can be greatly improved. Based on this idea GAIL
+[8] combines generative network GAN and maximum entropy IRL, which can be continuously and efficiently trained without the need to manually label expert data continuously.
+
+On this basis, many works have improved GAIL. eg InfoGail[9] replaced GAN with WGAN and achieved better results. There are also some recent works such as GoalGAIL [10], TRGAIL [11] and DGAIL [12] which combine other methods such as post-hoc relabeling and DDPG to achieve faster convergence and better final performance.
+
+
+Future Study
+--------
+
+There are still many challenges in current imitation learning, mainly including the following:
+
+- The current imitation learning is for a specific task, and there is a lack of imitation learning methods that can be applied to multiple tasks;
+
+- The current imitation learning algorithm is not optimal for expert data, and it is difficult to surpass expert data to achieve optimal results;
+
+- The current imitation learning algorithm is mainly aimed at observation, and cannot combine multi-modal factors such as speech and natural language;
+
+- Current imitation learning can find local optima, but often cannot find global optima.
```

Comment:
There are still many challenges in imitation learning, mainly including the following:

- Generally speaking, it is for a specific task, and there is a lack of imitation learning methods that can be applied to multiple tasks;

- For situations where the data is not optimal, it is difficult to surpass the data to achieve optimal results;

- Mainly focus on the research of observation, without combining multi-modal factors such as speech and natural language;

- The local optimum can be found, but the global optimum can often not be found.